src/app/components/feed/jh-pushtotalk-button.html

```diff
@@ -0,0 +1,10 @@
+<button ng-class="{'btn-danger': vm.hotkeyActive }" class="btn" ng-click="vm.click()" title="{{vm.titleText}}">
+  <span class="glyphicon glyphicon-bullhorn" aria-hidden="true">
+  </span>
+  <span ng-show="vm.showHotkey">
+    <kbd style="background-color:#f15a5c">
+        {{vm.hotkey}}
```

Comment:
You're right. I did now some stress testing and I see that it records the previous key. I also uncovered a second bug in my function abortKey, I need to add `if(_recordedSequenceCallback !== null)` because if aborting very fast before it finished triggering all callbacks it throws an error.